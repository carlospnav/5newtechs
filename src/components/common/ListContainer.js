import { flexGroup } from "theme/utils"
import styled from "styled-components"

export default styled.div`
  ${flexGroup("column")};
  padding: 0 20px;
`
