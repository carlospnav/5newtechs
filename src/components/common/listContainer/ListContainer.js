import { flexGroup } from "theme/utils"
import styled from "styled-components"

export default styled.div`
  position: relative;
  ${flexGroup("column", "center", "center")};
  padding: 0 20px;
  width:  1200px;
`