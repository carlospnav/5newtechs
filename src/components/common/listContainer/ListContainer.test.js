import { shallow } from "enzyme"
import ListContainer from "./ListContainer"
import "jest-styled-components"

describe("<ListContainer>", () => {
  it("should render a ListContainer with its children properly.", () => {
    const wrapper = shallow(
      <ListContainer>
        <h2>I'm a container!</h2>
        <p>Indeed I am!</p>
      </ListContainer>
    )
    expect(wrapper).toMatchSnapshot()
  })
})