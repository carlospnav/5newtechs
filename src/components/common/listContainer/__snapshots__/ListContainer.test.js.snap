// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListContainer> should render a ListContainer with its children properly. 1`] = `
.c0 {
  position: relative;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 0 20px;
  width: 1200px;
}

<div
  className="c0"
>
  <h2>
    I'm a container!
  </h2>
  <p>
    Indeed I am!
  </p>
</div>
`;
